storybook/storybook-react/src/components/Heading/Heading.stories.tsx

```diff
@@ -14,7 +14,17 @@ const meta = {
   args: {
     children: 'Jouw typograaf biedt mij zulke exquise schreven!',
     level: 1,
-    styleLevel: 1,
+    size: 'level-1',
+  },
+  argTypes: {
+    level: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
In the React story we do want to offer the `level` control, right?

Reply:
I copied this from the CSS Story. In both cases, we show separate pages for Heading 1 – 4. I don’t think it makes sense to allow changing the level for one of those if all levels have their own page.


We could combine all headings into one Story which does expose the `level` prop, but I think I prefer the current setup.

Reply:
For CSS stories the level prop is removed because the user has to generate the HTML tags themselves. For the React component the component does that for you. Generally, we show all controls with all options for a component in the first story, with specific settings of props in the subsequent stories. I think that makes sense, if you want to play with the different props you can do so in the first story.

Reply:
Alright, I’ve restored it.